modules/rock-paper-scissors/module.json

```
@@ -11,7 +11,7 @@
   },
   "description": {
     "en": "Let your users play Rock Paper Scissors against the bot and each other!",
-    "de": "Lasse Nutzer auf deinem Server Schere Stein Paper gegen den Bot und gegeneinander spielen"
+    "de": "Lasse Nutzer auf deinem Server Schere Stein gegen den Bot und gegeneinander spielen"
```

```suggestion
    "de": "Lasse Nutzer auf deinem Server Schere Stein Papier gegen den Bot und gegeneinander spielen"
```
*Paper left the chat*